behavior/MouseTracking/FindTrials/mouse_path_plotter.m

```diff
@@ -47,9 +48,12 @@
         set(gca,'XTick',[],'YTick',[]);
         colormap gray;
         hold on
-        plot(centroids(trial_start:trial_end,1),...
-             centroids(trial_start:trial_end,2),...
-             'g-');
+        
+        % plot centroids in color
+        for i = trial_start:trial_end
+            color_options = jet(size(centroids(trial_start:trial_end,1),1));
```

Comment:
Actually, this line should go outside of the `for i = 1:trial_start:trial_end` loop.